memeolist/app/src/main/java/org/aerogear/android/app/memeolist/ui/MemeListActivity.java

```diff
@@ -245,6 +229,14 @@ public void onException(Exception exception) {
                 });
     }
 
+    @BindingAdapter("ownerAvatar")
+    public static void displayOwnerAvatar(ImageView imageView, Meme meme) {
```

Comment:
we should add null checks on imageView and meme

Reply:
Well, It will never be null, but I don't care to add those checks.

Reply:
I believe it should be added at least an annotation.

Reply:
Like I said I really don't care to add but the annotation will not be used anywhere because it's by the Android system on the xml layout